### ISSUE TITLE
Point ‘help’ on the the Educators page to the Activities Page 

### ISSUE THREAD

**Issue by kaschm** [LLK/scratch-www#905]
scratch.mit.edu/educators 

In advance of CSEdWeek, from within the "Guides and Tutorials" section, we should point to the Activities page rather than to the Help Page. We may also want to update the image associated with the text. 

From: 
On the Help Page, you can find workshop guides, Scratch Cards, videos, and other resources.

To:
On the Activities Page, you can find ten themed activity packs for Scratch. Each activity pack comes with a tutorial, a set of helpful cards, and a facilitator guide. 

This is blocked on the creation of the Activity page: 
https://github.com/LLK/scratch-www/issues/904

**Comment by ntlrsk** [comment]
Currently there is no way on the Scratch site to get to the "Things to Try" page.
For the Educator page, it could keep the same design but just change the text: 
Under Guides & Tutorials, just change the text for the first two items:

* "The Things to Try page offers a variety of tutorials, activity cards, and educator guides."
* "The Tips Window provides help for creating projects in Scratch."

**Comment by ntlrsk** [comment]
@kaschm @mewtaylor I thought this was going to go in for HOC, and just saw it had been blocked. If there's anyway to just change the text, that would be great, so that educators can find the Things to Try page from our site. If not, I understand.

**Comment by chrisgarrity** [comment]
@kaschm @ntlrsk I believe that @mewtaylor is already on his way home for the holidays, but I could make changes to the Educator page. Just to confirm. The middle item should be:
tips-window icon: linked to ?tip_bar=home
The Things to Try page offers a variety of tutorials, activity cards, and educator guides. (with 'Things to Try page' linked to /go)
The Tips Window provides help for creating projects in Scratch.: (with 'Tips Window" linked as it is now
Just to note the current text is: "The Tips Window features step-by-step tutorials for getting started in Scratch."

**Comment by mewtaylor** [comment]
@kaschm @ntlrsk @chrisgarrity I'd recommend we put this in the November 21 hotfix, which will go out for HOC

**Comment by kaschm** [comment]
Thanks for catching this, @ntlrsk . I just moved it up to Nov 21st. @chrisgarrity , thanks for grabbing this!

**Comment by ntlrsk** [comment]
Yes, thanks for confirming and putting this in @chrisgarrity

**Comment by chrisgarrity** [comment]
@ntlrsk @kaschm should the text be separate paragraphs:
![Screen Shot 2016-11-23 at 9.41.59 AM.png](https://codetreeusercontent.s3.amazonaws.com/uploads/407f3a34-3371-488a-b531-8b75c21f215a/Screen+Shot+2016-11-23+at+9.41.59+AM.png)

or one combined paragraph:![Screen Shot 2016-11-23 at 9.40.51 AM.png](https://codetreeusercontent.s3.amazonaws.com/uploads/407f3a34-3371-488a-b531-8b75c21f215a/Screen+Shot+2016-11-23+at+9.40.51+AM.png)

**Comment by TheGrits** [comment]
@chrisgarrity If I may chip in my two cents I think that the two paragraphs looks better.

**Comment by chrisgarrity** [comment]
Sorry, for the additional images, but I realized the context on the [educator page](https://scratch.mit.edu/educators/) may influence opinions:

Two paragraph version:

<img width="1062" alt="screen shot 2016-11-23 at 9 57 29 am" src="https://cloud.githubusercontent.com/assets/399209/20566534/69962286-b163-11e6-88c8-d643f9657ea4.png">

Single combined paragraph:
<img width="1008" alt="screen shot 2016-11-23 at 9 56 35 am" src="https://cloud.githubusercontent.com/assets/399209/20566548/6eed7dd8-b163-11e6-8013-a5d7f5ff24e3.png">

I think I have a slight preference for the two paragraph version myself, but could go either way.

**Comment by TheGrits** [comment]
@chrisgarrity Would you consider just a new line to separate the two paragraphs, like so? 
![scratch mit edu-educators-](https://cloud.githubusercontent.com/assets/12952370/20566947/e4e5e5ba-b164-11e6-9cef-11eb72c57976.png)

**Comment by chrisgarrity** [comment]
@TheGrits good compromise, I'll go with that until @ntlrsk has input. Thanks.

**Comment by ntlrsk** [comment]
@chrisgarrity The Things to Try should replace the Help Page text.

**Comment by chrisgarrity** [comment]
Ok, here's what it looks like with that change:
<img width="1087" alt="screen shot 2016-11-23 at 10 20 18 am" src="https://cloud.githubusercontent.com/assets/399209/20567397/842b6734-b166-11e6-8964-eb453cc910de.png">

**Comment by ntlrsk** [comment]
Yes, that seems good for now. We can revisit after HOC, but for now at least provides a way to get to Things to Try. Thanks!

**Comment by kaschm** [comment]
@ntlrsk yup, we have another issue here to track that: https://github.com/LLK/scratch-www/issues/1038

**Comment by chrisgarrity** [comment]
Fixed in https://github.com/LLK/scratch-www/pull/1074
